SeaPublicWebsite.UnitTests/QuestionFlowServiceTests.cs

```diff
@@ -1365,36 +1404,73 @@ public void RunSkipLinkTestCases(QuestionFlowServiceTestCase testCase)
                 new { reference = "ABCDEFGH" }
             )),
         new(
-            "Accessible loft space continues to roof insulated if user has it",
+            "Loft space continues to accessible loft if user has it",
+            new Input(
+                QuestionFlowPage.LoftSpace,
+                "ABCDEFGH",
+                loftSpace: LoftSpace.Yes
+            ),
+            new PathByActionArguments(
+                nameof(EnergyEfficiencyController.AccessibleLoft_Get),
+                "EnergyEfficiency",
+                new { reference = "ABCDEFGH" }
+            )),
+        new(
+            "Changing loft space continues to answer summary if user does not have loft space",
+            new Input(
```

Comment:
What about "Changing loft space continues to loft access if user has loft space"

Please have a think about all the possible routes here and make sure they are covered